x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -1841,6 +1817,30 @@
         }
       }
     },
+    "infraops": {
```

Comment:
@elastic/kibana-telemetry Out of curiosity, any idea why the order of fields are changing here? Would be nice if we could make it stable to avoid unnecessary diffs.

Reply:
@Bamieh do you have any idea how we can prevent reordering fields in the json files?